brave-tracesamplers/pom.xml

```diff
@@ -7,9 +7,9 @@
     <version>3.3.1-SNAPSHOT</version>
   </parent>
   
-  <artifactId>brave-tracefilters</artifactId>
+  <artifactId>brave-tracesamplers</artifactId>
   <packaging>jar</packaging>
-  <name>brave-tracefilters</name>
+  <name>brave-tracesamplers</name>
```

Comment:
seems brave-tracesampler-zookeeper follows more similarly to
brave-spancollector-scribe (lowercase type name-impl), so going with
that..